### PR TITLE
Lhmc CollectionObject

### PR DESCRIPTION
**Pull request by kspurgin**
Sets up a pattern for extension modules that are usable across data profile modules. Creates lhmc collectionobject converter and adds CulturalCare extension to both lhmc and anthro.

### CODE REVIEW

**Comment by mark-cooper** [comment]
@kspurgin I made a few updates and changed up how the extension piece works here:

https://github.com/collectionspace/cspace-converter/pull/297

I think this is a simpler starting point for future refactoring (I'd like to get away from all the class methods eventually), and I don't think it feels right for an extension to be a sub-class of a record type because they're not CRUD-able directly.

**Comment by kspurgin** [comment]
OK, merging in the classless module idea from #297 but keeping a separate module for each extension.

File: lib/collectionspace/converter/default/record.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module CollectionSpace
   module Converter
     module Default
+      ::Default = CollectionSpace::Converter::Default
```
Comment (mark-cooper) [inline lib/collectionspace/converter/default/record.rb:4]: I think we can remove this line. If there's a weird load order issue we can use require_relative to include default.rb first.

**Comment by mark-cooper** [comment]
@kspurgin just asked for one trivial update, otherwise LGTM 👍

File: lib/collectionspace/converter/default/record.rb (hunk 1 above)
Comment (kspurgin) [reply lib/collectionspace/converter/default/record.rb:4]: Ok, wasn't sure what that did, but just copied it over from your alternate PR. 😆

**Comment by kspurgin** [comment]
Oh and that default line may have originally been in your alternate PR because for like 15 minutes at some point, I think I thought I needed it for a reason that escapes me.

**Pull request merged**